general-superstaq/general_superstaq/superstaq_client.py

```diff
@@ -402,7 +402,12 @@ def _handle_status_codes(self, response: requests.Response) -> None:
             else:
                 message = str(response.text)
             raise gss.SuperstaQException(
-                f"Non-retriable error making request to SuperstaQ API, {message}",
+                f"""Non-retriable error making request to SuperstaQ API, {message}.
+
+                If you would like to contact a member of our team,
+                email us at superstaq@infleqtion.com or join our Slack workspace:
+                https://join.slack.com/t/superstaq/shared_invite/zt-1wr6eok5j-fMwB7dPEWGG~5S474xGhxw
+                """,
```

Comment:
this will have weird line breaks/indents. it's better to use single quotes to break up long strings:
```suggestion
            slack_invite_url = (
                "https://join.slack.com/t/superstaq/shared_invite/"
                "zt-1wr6eok5j-fMwB7dPEWGG~5S474xGhxw"
            )
            raise gss.SuperstaQException(
                f"Non-retriable error making request to SuperstaQ API, {message}.\n\n"
                "If you would like to contact a member of our team, email us at "
                f"superstaq@infleqtion.com or join our Slack workspace: {slack_invite_url}",
```
(the slack url could also be placed inline, but seemed nice to have it as its own variable as it's subject to change)